### ISSUE TITLE
How can I improve the output with my custom video??

### ISSUE THREAD

**Issue by Samuel3Shin** [BGU-CS-VIL/DeepMCBM#2]
I get some bad output from my video.

I think the model makes incorrect panoramic images, therefore the final results are not good as well...

The panoramic_robust_mean.png are like below.
![panoramic_robust_mean](https://user-images.githubusercontent.com/47931506/205424469-3f066e34-c63a-4326-ab88-bb1e3d3c3e55.png)
![panoramic_robust_mean](https://user-images.githubusercontent.com/47931506/205424479-476dbbf2-d6d7-4aaa-a4b0-c7e969cc681f.png)

Is there some way to improve outputs from my custom videos by manipulating the hyperparameter or something??

Thanks!!!

P.S
Those are my run command, so I used all the default arguments.
python src/DeepMCBM.py --dir video1
python src/DeepMCBM.py --dir video2

**Comment by guy-erez** [comment]
You should consider reducing the learning rate. It is highly recommended to track the alignment status using some asynchronous logging mechanism. A version that supports Neptune.ai may be released in the future.
